client.go

```diff
@@ -136,6 +136,7 @@ type Packet struct {
 	Logger    string    `json:"logger"`
 
 	// Optional
+	Release    string                 `json:"release,omitempty"`
```

Comment:
Nitpick: I like to keep the fields in the same order as the [Writing a Client Guide](http://sentry.readthedocs.org/en/latest/developer/client/#release).

```go
Platform   string                 `json:"platform,omitempty"`
Culprit    string                 `json:"culprit,omitempty"`
ServerName string                 `json:"server_name,omitempty"`
Release    string                 `json:"release,omitempty"`
Tags       Tags                   `json:"tags,omitempty"`
Modules    []map[string]string    `json:"modules,omitempty"`
Extra      map[string]interface{} `json:"extra,omitempty"`
```

Rationale: double checking we have all the fields is easier. I think server name may have moved in the docs, or maybe it was always wrong, but you don't have to worry about that in this PR if you don't want to.